src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -327,6 +329,29 @@ public static FormulaValue Text(EvalVisitor runner, EvalVisitorContext context,
             return Text(runner.GetFormattingInfo(), irContext, args, runner.CancellationToken);
         }
 
+        public static FormulaValue TextToBlob(EvalVisitor runner, EvalVisitorContext context, IRContext irContext, FormulaValue[] args)
+        {
+            runner.CancellationToken.ThrowIfCancellationRequested();
+
+            StringValue sv = (StringValue)args[0];           
+            return new BlobValue(new StringBlob(sv.Value));

```

Comment:
this is contents (not uri)?  Should this fail completely?

Reply:
See AzureBlobStorage.CreateFile example